openmls/src/treesync/mod.rs

```diff
@@ -627,26 +712,4 @@ mod test {
     fn test_ratchet_tree_internal_empty_after_trim() {
         RatchetTree::trimmed(vec![None]);
     }
-
-    #[test]
-    fn test_ratchet_tree_trailing_blank_nodes() {
```

Comment:
Why was this removed? I remember there was a discussion on trailing blank nodes, but I can't remember the outcome. Are they allowed now?

Reply:
It's back now below.